chia/_tests/blockchain/test_blockchain.py

```diff
@@ -152,7 +152,14 @@ async def test_overflow_genesis_empty_slots(self, empty_blockchain: Blockchain,
         await _validate_and_add_block(empty_blockchain, genesis)
 
     @pytest.mark.anyio
-    async def test_genesis_validate_1(self, empty_blockchain: Blockchain, bt: BlockTools) -> None:
+    async def test_genesis_validate_1(
+        self, empty_blockchain: Blockchain, bt: BlockTools, monkeypatch: pytest.MonkeyPatch
+    ) -> None:
+        # Monkey patch prev_tx_block so we dont throw there
```

Comment:
I don't understand why we need to monkeypatch this here. Isn't it safe to call `get_consecutive_blocks()` without throwing?
I think it would be good for this comment to explain what's special about this call.